auction9-backend/handler.js

```diff
@@ -250,3 +252,46 @@ export const realizeFinishedAuction = async (event, context) => {
     });
   }
 };
+
+/* postNewBid - creates new auction
+ * POST: /auctions/id/new
+ */
+export const postNewBid = async (event, context) => {
+  try {
+    let reqBody = JSON.parse(event.body);
+    let newBid = reqBody.newBid;
+    let currentAuctionPrice = await mysql.query('SELECT price FROM tbl_auction WHERE auctionID=?', [reqBody.auction.auctionID]);
+    await mysql.end();
+    // checking if newbid is greater then current price
+    if (newBid > currentAuctionPrice[0].price) {
+      // date formatting
+      let bidDate = Date.now();
+      let todayDate = new Date(bidDate);
+      let currentYear = todayDate.getFullYear();
+      let currentMonth = todayDate.getMonth() + 1;
+      let currentDay = todayDate.getDate();
+      let currentHours = todayDate.getHours();
+      let currentMinutes = todayDate.getMinutes();
+      let currentSeconds = todayDate.getSeconds();
+      let formattedTodayDate = currentYear + '-' + currentMonth + '-' + currentDay + ' ' + currentHours + ':' + currentMinutes + ':' + currentSeconds;
+      // first create bid -> update current price with new bid
+      // current userid hardcoded
+      await mysql.query('INSERT INTO tbl_user_auction (`userID`, `auctionID`, `price`, `time`) VALUES (?, ?, ?, ?)', [2, reqBody.auction.auctionID, newBid, formattedTodayDate]);
+      await mysql.end();
+      await mysql.query('UPDATE tbl_auction SET price=? WHERE auctionID=?', [newBid, reqBody.auction.auctionID]);
+      await mysql.end();
```

Comment:
`mysql.end()` should be called only once in the end

Reply:
Okay, other calls removed